Example/GiniVision/Base.lproj/Main.storyboard

```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="10117" systemVersion="15G1004" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" initialViewController="Q18-fR-y4t">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This change seems to be just Interface Builder being silly and changing version stuff. I'd prefer if we discarded these changes unless we have something meaningful to change in the storyboard.